### PR TITLE
[RAPPS] Cleanup headers inclusions.

### PR DESCRIPTION
**Pull request by HBelusca**
## Purpose

Looking at the source of RAPPS gives the impression that a lot of headers are included again and again in the different files (or some are included when they are not needed, because they are included via another file, etc.).
Let us "slap some sense" in these headers.

### CODE REVIEW

**Comment by HBelusca** [comment]
Cc @kernelbin too (I couldn't add you to the list of reviewers for unknown reasons...)

File: base/applications/rapps/include/winmain.h
Comment (binarymaster) [inline base/applications/rapps/include/winmain.h:3]: WIP leftover?

**Review by binarymaster — APPROVED** [review]
:white_check_mark: CI status is green.

File: base/applications/rapps/include/winmain.h
Comment (HBelusca) [reply base/applications/rapps/include/winmain.h:3]: The PR was also a pretext for suggesting which other headers could be factored out / in which files it makes sense to do that.
And yes, this is some leftover.

**Review by JoachimHenze — APPROVED** [review]

**Comment by learn-more** [comment]
As said before in the chat, I'd prefer if you did not merge this, since I am restructuring rapps a bit.

**Comment by HBelusca** [comment]
> As said before in the chat, I'd prefer if you did not merge this, since I am restructuring rapps a bit.

Noted, thanks for the information!

**Comment by learn-more** [comment]
> @learn-more Did you finalize your RAPPS restructuring ? Should this PR be kept in stand by ? canceled ? Moved on ?

I did do one round of cleanup, and won't be doing any more in the foreseeable future.

**Comment by KRosUser2** [comment]
@HBelusca : up to you !

**Review by kernelbin — APPROVED** [review]
Looks good for me

**Pull request merged**
